.goreleaser.yml

```diff
@@ -12,6 +12,12 @@ release:
     - glob: ./manifest/charts/*
     - glob: ./manifest/yamls/*
 
+variables:
+  fetcherDescription: Fetcher is a very lightweight component and all of related logic is in fetcher package itself. Fetcher helps in fetching and uploading code and in specializing environments.
+  builderDescription: Builder builds the source code into executable binary files.
+  reporterDescription: Reporter reports fission events to analytics service.
```

Comment:
The reporter gathers information that assists in improving fission.

Reply:
Done